Resources/Prototypes/explosion.yml

```diff
@@ -84,3 +84,19 @@
   lightColor: Orange
   texturePath: /Textures/Effects/fire.rsi
   fireStates: 3
+
+- type: explosion
+  id: PowerSink
+  damagePerIntensity:
+    types:
+      Heat: 12
+      Blunt: 12
+      Piercing: 12
+      Structural: 40
+  tileBreakChance: [ 0, 0.5, 1 ]
+  tileBreakIntensity: [ 1, 5, 10 ]
+  tileBreakRerollReduction: 3
+  intensityPerState: 20
+  lightColor: Orange
+  texturePath: /Textures/Effects/fire.rsi
+  fireStates: 6
```

Comment:
Check with electro but pretty sure every new explosion type added is more data added to the explosion map for every single tile when this could just re-use an existing one.

Reply:
.... did you read my note above about Explosion Type?

Reply:
Are you referring to:

> Using the new explosion type, this should work even if the wall is reinforced, encouraging tricky things like walling them up near important objectives.

Every additional explosion type adds to overhead on tracking it for the explosion tiles so if preferable would be better to be shared with minibomb / microbomb / demolition.

Reply:
I was referring to:

> Explosion Type - With regards to new explosion type, no, the previous types did not accomplish the wall breaching and into the next room very well during my tests. Generally speaking the reinforced walls would block the explosion entirely with those. This is why I added a new type, so I could get this effect without having to boost intensities ultra high.

Reply:
(it is possible to get this effect by boosting intensities super high, but I got laughed at in discord for doing this before)